### PR TITLE
Enable SDK Automation V2 in CI

### PR DESCRIPTION
**Pull request by PhoenixHe-NV**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -11,14 +15,23 @@ variables:
...
-- template: .azure-pipelines/Syntax.yml
-- template: .azure-pipelines/Semantic.yml
-- template: .azure-pipelines/Avocado.yml
-- template: .azure-pipelines/ModelValidation.yml
-- template: .azure-pipelines/BreakingChange.yml
-- template: .azure-pipelines/LintDiff.yml
-- template: .azure-pipelines/SwaggerToSDK.yml
-- template: .azure-pipelines/NetworkValidation.yml
-- template: .azure-pipelines/Spellcheck.yml
+stages:
```
Comment (yungezz) [inline azure-pipelines.yml]: what's difference between stage and job

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
looks good in general, about when to enable it, need to do it after Ignite, Nov 4th to avoid any impact to Ignite asks.

File: azure-pipelines.yml (hunk 1 above)
Comment (NullMDR) [reply azure-pipelines.yml]: stage are set of jobs that could be triggered together with certain condition or work flow.

**Comment by NullMDR** [comment]
@ArcturusZhang @erich-wang @isra-fel @zikalino @fengzhou-msft @yaohaizh for awareness

**Comment by yungezz** [comment]
i've approved this PR, seems CI status not updated

**Comment by yungezz** [comment]
loop @akning-ms for review also

**Pull request merged**